detection_rules/rule_validators.py

```diff
@@ -108,6 +112,13 @@ def validate(self, data: 'QueryRuleData', meta: RuleMeta) -> None:
                 print(err_trailer)
                 raise
 
+            if endgame_schema:
+                try:
+                    with endgame_schema:
+                        eql.parse_query(self.query)
+                except eql.EqlParseError as exc:
+                    print(str(exc))
+
```

Comment:
@brokensound77 We should discuss the validation here. I recall you mentioned saying we should reuse the error message from the previous check.